sourcemash/frontend/templates/category.tpl

```diff
@@ -7,11 +7,25 @@
 
   <div class="loading center-align"></div>
 
-  <ul id="items" class="list-group row">
-    <% items.forEach(function(item) { %>
-        <div id="item-<%= item.get('id') %>" class="item-card"></div>
-    <% }); %>
-  </ul>
+  <div id="items" class="list-group row">
+    <div class="col m6 hide-on-small-and-down">
+      <% for (i = 0; i < items.length; i = i + 2) { %>
```

Comment:
Ah I see how you split the two columns. I was imagining one loop with two div's inside it that alternated indices with if statements. Looks good!